lib/ecto_ordered.ex

```diff
@@ -34,23 +34,16 @@ defmodule EctoOrdered do
 
   """
 
+  @max 8388607
+  @min -8388607
```

Comment:
It was a hold-over from https://github.com/mixonic/ranked-model, but I'm thinking a BIGINT is going to be a better choice in the long term.